doc/ReleaseNotes/develop.toml

```diff
@@ -51,7 +51,7 @@ description = "Fixed a memory access exception that can occur when writing conve
 [[items]]
 section = "features"
 subsection = "solution"
-description = "Added an optional new COORDINATE\\_CHECK\\_METHOD option to the PRT model's Particle Release Package, accepting values 'EAGER' (default) or 'NONE'. 'NONE' disables release-time verification that release point coordinates and release point cell IDs match, which can significantly reduce runtime when releasing many particles."
```

Comment:
retrospective fix, just noticed that we don't use quotes around option values in release notes